### PR TITLE
Made focused discovery from description card (in map URL) pin's bigger

### PR DESCRIPTION
**Pull request by SarahHCode**
# Pull Request

## Summary
Added code to make focused pin bigger. 

## Changes Made
Added the code in showPins() method to make the focused discovery in URL pin bigger and make former focused pin (previously in URL) return to normal style (because setStyle() overrides or something the normal style). Also added a new variable to keep track of former focused pin. Added showPins() method to $watch in created() cycle to watch for changes in focused discovery in URL to update bigger pin. 💀

## Checklist
Please ensure all the following steps are completed before submitting the pull request:

- [ ] Code passes all existing tests
- [ ] New features or changes are accompanied by appropriate tests
- [ ] Code follows the established coding style and conventions
- [ ] Documentation has been updated to reflect the changes (if applicable)
- [ ] All new and existing tests pass locally

## Additional Notes
How do I make tests for this? Plus, I am scared that with how many times showPins() is called, that the performance would suffer, so, I think that should be checked.

### CODE REVIEW

File: src/components/MapContainer.vue [hunk 1]
```diff
@@ -47,17 +47,18 @@ import { useRoute } from "vue-router";
 import { Icon, Style } from "ol/style";
 import customLocationIcon from "@/assets/drawable/icons/location.svg";
 //TODO: find out why images are not displayed
-// This variable is here to know if the user focuses a discovery or not
+// This variable is here to know if the user focuses (previous click is) on a discovery or not
 let hasFocus = false;
 
-function insertAllPins(destination, list) {
-  for (const discovery of list) {
+//TODO figure out why this method is here
```
Comment (kim841) [inline src/components/MapContainer.vue:53]: Il y a un mélange entre option et composition api

**Review by kim841 — APPROVED** [review]

**Comment by kim841** [comment]
Overall, très bon code! Les commentaires sont très adéquats!

As-tu eu des crash de l'application ? J'en ai eu 2 fois. Quand j'ai recommencé l'émulateur, c'était smooth. 

Pour les tests, le focus du pin arrive lors de l'intéraction entre 2 écrans (ex: fiche descriptive d'un discovery & la map), e2e pourrait être une manière de tester

**Comment by SarahHCode** [comment]
Oui, j'ai eu des crashs... je me demande si ce serait à corriger...

**Comment by SarahHCode** [comment]
À prioriser plus tard a cause des crashs

**Pull request merged**